### ISSUE TITLE
MCQ form isn't valid, improper use of fieldset and legend (WCAG 1.3.1) [QAC]

### ISSUE THREAD

**Issue by ftourangeau** [MeD-DMC/Adapt-accessibilityfixes#211]
**COURSE:**
https://esdc-edsc-adapt.canadacentral.cloudapp.azure.com/preview/5dc446fd27f08836496b6b9e/63a32079b75e454dafed1e1d/
*******************************************************************************
**Page:**
Lesson 1: Overview of electronic Passport Next Generation
Lesson 2: Domestic Print
Lesson 3: Observation Print
Lesson 4: Manual QA
Lesson 5: Reports

**Description:**
Improper use of the legend element used, legend cannot be a child of a div element. Occurs with each question.

Legend must be the first child of <fieldset> tag

i.e. <fieldset>
      <legend> </legend>
       <div> </div>
       </fieldset>

Please review the structure of the fieldset and legend.
![image](https://user-images.githubusercontent.com/93537804/229854709-9a68214a-d593-4877-b65c-4d53c8be910f.png)
*******************************************************************************
